idunn/blocks/wikipedia.py

```diff
@@ -5,7 +5,7 @@
 
 from apistar import validators
 from .base import BaseBlock, BlocksValidator
-from idunn.utils.prometheus import PrometheusTracker
+from idunn.utils.prometheus import wiki_request_duration, breaker_error, exception, limiter_exception
```

Comment:
I think it's better to only import the module and use it:
`prometheus.wiki_request_duration`

Reply:
done